tests/std/tests/P0896R4_common_iterator/test.cpp

```diff
@@ -311,6 +311,31 @@ constexpr bool test_lwg_3574() {
     return true;
 }
 
+// Validate that _Variant works when fed with a non-trivially-destructible type
+struct non_trivially_destructible_input_iterator {
+    constexpr ~non_trivially_destructible_input_iterator() {}
+
+    using difference_type = int;
+    using value_type      = int;

```

Comment:
(Pure style, so ignore if you like) type definitions typically precede functions in a class definition absent any reason to do otherwise.

Reply:
Looks like both of these style comments apply to other tests as well. I'm not requiring a change, but if you do please change them all consistently.